Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -0,0 +1,123 @@
+require "utils/analytics"
+
+describe Utils::Analytics do
+  describe "::os_prefix_ci" do
+    context "when anonymous_os_prefix_ci is not set" do
+      it "returns OS_VERSION and prefix when HOMEBREW_PREFIX is not /usr/local" do
+        h_prefix = HOMEBREW_PREFIX
+        HOMEBREW_PREFIX = mktmpdir/"blah"
```

Comment:
It may be needed here, since it has to be in another directory than the default. Anyways, use [`stub_const`](https://relishapp.com/rspec/rspec-mocks/v/3-6/docs/mutating-constants/stub-defined-constant) instead of redefining it.